provisioning/stream/kubernetes/deploy/api/.gitignore

```diff
@@ -1,2 +0,0 @@
-*
-!.gitignore
```

Comment:
tohle tu imho mělo zůstat, sem se to je kopíruje pro deploy ne?

Reply:
[.gitignore](https://github.com/keboola/keboola-as-code/blob/michaljurecko-PSGO-591-release/provisioning/stream/kubernetes/.gitignore) tam je pre cely `deploy` adresar, len sa omylom commitli subory, pred upravou gitignore.

Fixnute: https://github.com/keboola/keboola-as-code/pull/1860/commits/95eb604d5cd72ed687309e5c1c504f27725fef1d